forge/routes/api/project.js

```diff
@@ -1320,6 +1320,49 @@ module.exports = async function (app) {
         })
     })
 
+    app.post('/:instanceId/update-state', {
+        preHandler: (request, reply, done) => {
+            // check accessToken is project scope
+            // (ownerId already checked at top-level preHandler)
+            if (request.session.ownerType !== 'project') {
+                reply.code(401).send({ code: 'unauthorized', error: 'unauthorized' })
+            } else {
+                done()
+            }
+        },
+        schema: {
+            summary: 'Update the live status of an instance',
```

Comment:
```suggestion
            summary: 'Update the live status of an instance',
            hide: true, 
```

This isn't an end-user API so shouldn't be part of our published API doc